src/minisakai.ts

```diff
@@ -370,43 +370,71 @@ function deleteFavoritesBarNotification(): void {
+  for (let i = 0; i < element.length; i++) {
+    const elem = element[i] as HTMLElement;
+    elem.setAttribute("style", "color:" + color + "!important");
+  }
+};
+
 /**
  * Override CSS of favorites bar and miniSakai.
  */
 async function overrideCSSColor() {
   const config = await loadConfigs();
-  const overwriteborder = function (className: string, color: string | undefined) {
-    const element = document.getElementsByClassName(className);
-    for (let i = 0; i < element.length; i++) {
-      const elem = element[i] as HTMLElement;
-      const attr = "solid 2px " + color;
-      (<any>elem.style)["border-top"] = attr;
-      (<any>elem.style)["border-left"] = attr;
-      (<any>elem.style)["border-bottom"] = attr;
-      (<any>elem.style)["border-right"] = attr;
-    }
-  };
-  const overwritebackground = function (className: string, color: string | undefined) {
-    const element = document.getElementsByClassName(className);
-    for (let i = 0; i < element.length; i++) {
-      const elem = element[i] as HTMLElement;
-      elem.setAttribute("style", "background:" + color + "!important");
-    }
-  };
-
+  
   // Overwrite colors
   overwritebackground("cs-course-danger", config.CSsettings.getMiniColorDanger);
   overwritebackground("cs-course-warning", config.CSsettings.getMiniColorWarning);
   overwritebackground("cs-course-success", config.CSsettings.getMiniColorSuccess);
   overwritebackground("cs-tab-danger", config.CSsettings.getTopColorDanger);
   overwritebackground("cs-tab-warning", config.CSsettings.getTopColorWarning);
   overwritebackground("cs-tab-success", config.CSsettings.getTopColorSuccess);
+  
   overwriteborder("cs-assignment-danger", config.CSsettings.getMiniColorDanger);
   overwriteborder("cs-assignment-warning", config.CSsettings.getMiniColorWarning);
   overwriteborder("cs-assignment-success", config.CSsettings.getMiniColorSuccess);
   overwriteborder("cs-tab-danger", config.CSsettings.getTopColorDanger);
   overwriteborder("cs-tab-warning", config.CSsettings.getTopColorWarning);
   overwriteborder("cs-tab-success", config.CSsettings.getTopColorSuccess);
+  
+}
+
+function overrideCSSDarkTheme(){
+  if(getSakaiTheme() == 'dark'){
+    let foregroundColorDark = "#D4D4D4";
+    let backgroundColorDark = "#555555";
+    let dateColorDark = "#e07071";
+    overwritebackground("cs-minisakai", backgroundColorDark);
+    overwritecolor("cs-assignment-time", foregroundColorDark);
+    overwritecolor("cs-assignment-date", dateColorDark);
+    overwritecolor("cs-quiz-time", foregroundColorDark);
+    overwritecolor("cs-minipanda", foregroundColorDark);
+    overwritecolor("cs-settings-tab", foregroundColorDark);
+    overwritecolor("cs-memo-item", foregroundColorDark);
+    overwritecolor("cs-minisakai-list", foregroundColorDark);
+    overwritecolor("cs-assignment-title", foregroundColorDark);
+    overwritecolor("cs-noassignment-p", foregroundColorDark);
+  }
```

Comment:
[nits]
バーの色も見にくいからこうするといいかも
```suggestion
    overwritecolor("cs-tab-danger", backgroundColorDark);
    overwritecolor("cs-tab-warning", backgroundColorDark);
    overwritecolor("cs-tab-success", backgroundColorDark);
  }
```